scout/server/blueprints/cases/controllers.py

```diff
@@ -178,20 +183,21 @@ def _populate_case_groups(store, case_obj, case_groups, case_group_label):
             case_group_label[group] = store.case_group_label(group)
 
 
-def _get_partial_causatives(store: MongoAdapter, case_obj: Dict) -> List[Dict]:
-    """Return any partial causatives a case has, populated as causative objs.
-    Args:
-        store(adapter.MongoAdapter)
-        case_obj(models.Case)
-    Returns:
-        partial(list(dict))
+def _get_partial_causatives(store: MongoAdapter, institute_obj: dict, case_obj: dict) -> List[Dict]:
+    """Check for partial causatives and associated phenotypes.
+    Return any partial causatives a case has, populated as causative objs.
     """
 
     partial_causatives = []
     if case_obj.get("partial_causatives"):
         for var_id, values in case_obj["partial_causatives"].items():
+            variant_obj = store.variant(var_id)
+            if variant_obj:
+                decorated_variant_obj = _get_decorated_var(
```

Comment:
The causatives/suspects were not "decorated" or "parsed" so did not have e.g. the `str_mc` field set to display.